website/docs/docs/dbt-cloud-apis/sl-jdbc.md

```diff
@@ -215,21 +215,33 @@ select * from {{
 
 ### Query with where filters
 
-Where filters have three components: 
+Where filters in API allow for a filter list or string. We recommend using the filter list for production application as this format will realize all benefits from predicate pushdown where possible. 
```

Comment:
@rpourzand , what does predicate mean in this sentence?

```suggestion
In the API, you can use where filters either as a filter list or string. For production applications, it's better to use a filter list. This format helps you take advantage of all the benefits from the predicate pushdown where possible.

Where filters in API allow for a filter list or string. We recommend using the filter list for production application as this format will realize all benefits from predicate pushdown where possible. 
```

Reply:
So a predicate is just an expression to determine what rows in a database apply to a particular query. For example, if I filter in a WHERE clause on a particular dimension value, the database is basically going to search to determine what values in the DB apply to this query. If this can be pushed down to the database (predicate pushdown)), it's much faster.

Reply:
We could link out some generic docs on predicate pushdown.

Reply:
Predicate Pushdown points to the where or filter clause which affects the number of rows returned. It basically relates to which rows will be filtered, not which columns. For this reason, while applying the filter on a nested column as ‘library.books’ to merely return records with the values that are not null, the predicate pushdown function will make parquet read blocks that contain no null values for the specified column.